api/src/main/java/org/openmrs/module/isanteplusreports/report/renderer/IsantePlusIndicatorReportHtmlReportRenderer.java

```diff
@@ -0,0 +1,108 @@
+package org.openmrs.module.isanteplusreports.report.renderer;
```

Comment:
@hmutaquiha This whole class is not being referenced anywhere, please remove it.